### PR TITLE
Ability to favorite events and view favorited events on user profile page

### PR DESCRIPTION
**Pull request by cmaija**

### CODE REVIEW

File: parkhangs-expressapp/controllers/userController.js [hunk 1]
```diff
@@ -89,7 +90,7 @@ const updateUser = async (req, res) => {
 
         if (body.savedParks !== undefined) {
             try {
-                await savePark(user.savedParks, body.savedParks)
+                await saveParks(user.savedParks, body.savedParks)
             } catch (error) {
                 console.error(error.message)
                 console.log("unable to update park favourite count")
```
Comment (gordonzhang17) [inline parkhangs-expressapp/controllers/userController.js:96]: remove the 2 console statements

File: parkhangs-expressapp/controllers/userController.js [hunk 2]
```diff
@@ -102,6 +103,21 @@ const updateUser = async (req, res) => {
             user.savedParks = body.savedParks
         }
 
+        if (body.savedEvents !== undefined) {
+            try {
+                await saveEvents(user.savedEvents, body.savedEvents)
+            } catch (error) {
+                console.error(error)
+                console.log('unable to update event favorite count')
+                return res.status(404).json({
```
Comment (gordonzhang17) [inline parkhangs-expressapp/controllers/userController.js:112]: remove the 2 console statements

File: parkhangs-expressapp/controllers/userController.js (hunk 1 above)
Comment (cmaija) [reply parkhangs-expressapp/controllers/userController.js:96]: these are so we can see the error in the console... necessary

File: parkhangs-expressapp/controllers/userController.js (hunk 2 above)
Comment (cmaija) [reply parkhangs-expressapp/controllers/userController.js:112]: again, necessary for error debugging and common practice.

**Pull request merged**